### PR TITLE
Return SIIP-Examples tutorials to PSY Docs

### PR DESCRIPTION
**Pull request by rodrigomha**
Couldn't rebase the other branch (probably because of master/main issues), so created a new one.

The example about US systems was removed.
I also moved the dynamic data tutorial to PSID.

Also removed some Literate examples directly with the markdown.

Preview: https://nrel-sienna.github.io/PowerSystems.jl/previews/PR992/

### CODE REVIEW

**Comment by rodrigomha** [comment]
@claytonpbarrows Once is done, the preview is here:
https://nrel-sienna.github.io/PowerSystems.jl/previews/PR992/

File: docs/src/tutorials/test_data/OMIB.raw
Comment (jd-lara) [inline docs/src/tutorials/test_data/OMIB.raw:1]: Just a minor comment here to call the folder `tutorials_data`

File: docs/table_code.jl
Comment (jd-lara) [inline docs/table_code.jl:1]: Does this file work?

**Comment by jd-lara** [comment]
In the preview looks like some portions meant to be hidden are showing 

image

File: docs/src/modeler_guide/system_dynamic_data.md [hunk 1]
```diff
@@ -3,10 +3,10 @@
 You can access example data in the [Power Systems Test Data Repository](https://github.com/NREL-Sienna/PowerSystemsTestData)
```
Comment (jd-lara) [inline docs/src/modeler_guide/system_dynamic_data.md:3]: There is no Utilsdata any more 

image

**Comment by rodrigomha** [comment]
It seems that the artifact in tabular data is not working in the preview. Need to debug this tomorrow since it is working locally.

**Comment by rodrigomha** [comment]
This looks ready for merge.

**Comment by jd-lara** [comment]
we should move this one to PSB https://nrel-sienna.github.io/PowerSystems.jl/previews/PR992/tutorials/powersystembuilder/#psb

**Comment by rodrigomha** [comment]
> we should move this one to PSB https://nrel-sienna.github.io/PowerSystems.jl/previews/PR992/tutorials/powersystembuilder/#psb

I put in hide some of the file directories to clean up the docs. I think for now is okay to leave the PSB tutorial here, because there is not really a good existing docs for PSB. Once we start doing that later we can move the md file there.

**Review by jd-lara — APPROVED** [review]

**Pull request merged**